### PR TITLE
Add test cases of connection

### PR DESCRIPTION
**Pull request by wangting0128**
Signed-off-by: wangting0128 <ting.wang@zilliz.com>

### CODE REVIEW

File: tests20/python_client/common/common_type.py [hunk 1]
```diff
@@ -68,6 +70,7 @@
 
 class CheckParams:
     """ The name of the method used to check the result """
+    false = False
```
Comment (yanliang567) [inline tests20/python_client/common/common_type.py:73]: what is this false for?

Comment (wangting0128) [reply tests20/python_client/common/common_type.py:73]: check the res of API request is false

File: tests20/python_client/check/func_check.py [hunk 2]
```diff
@@ -17,11 +18,14 @@ def __init__(self, res, func_name, check_res, check_params, **kwargs):
...
-        check_result = None
+        check_result = True
 
         if self.check_res is None:
-            pass
-            # log.info("self.check_res is None, the response of API: %s" % self.res)
+            assert self.check_res_result is True
+            check_result = True
+        elif self.check_res == CheckParams.false:
+            assert self.check_res_result is False
```
Comment (yanliang567) [inline tests20/python_client/check/func_check.py:27]: I don;t think this assert is neccessary

**Review by yanliang567 — APPROVED** [review]

**Pull request merged**